aws_lambda_powertools/utilities/parser/envelopes/sqs.py

```diff
@@ -34,8 +34,10 @@ def parse(self, data: Dict[str, Any], model: Union[BaseModel, str]) -> List[Unio
         List
             List of records parsed with model provided
         """
+        logger.debug(f"Parsing incoming data with SQS model {SqsModel}")
         parsed_envelope = SqsModel(**data)
         output = []
+        logger.debug(f"Parsing SQS records in `body` with {model}")
         for record in parsed_envelope.Records:
             output.append(self._parse(record.body, model))
```

Comment:
self._parse can not get `str` type as a model argument.

Reply:
Reverted to kwargs to avoid confusion - `output.append(self._parse(data=record.body, model=model))`.